crate/kmip/src/crypto/elliptic_curves/kmip_requests.rs

```diff
@@ -65,6 +65,7 @@ fn build_mask_from_curve(
 /// - NIST.SP.800-186 - Section 3.1.2 table 2.
 /// - NIST.FIPS.186-5
 #[cfg(not(feature = "fips"))]
+#[allow(clippy::unnecessary_wraps)]
```

Comment:
do we need this ?

Reply:
actually, we do need this du to the signature function in fips mode